app/characterization_services/mediainfo_characterization_service.rb

```diff
@@ -152,10 +153,6 @@ def file_object
     # Retrieves the master binary file in this FileSet
     # @return [FileNode]
     def preservation_file
-      if parent.try(:image_resource?)
-        @file_set.original_file
-      else
-        @file_set.preservation_file
-      end
+      @file_set.original_file
```

Comment:
So wait, A/V bags don't use this characterizer?

Reply:
I'm still investigating this, but for the case with https://github.com/pulibrary/figgy/blob/master/spec/jobs/ingest_archival_media_bag_job_spec.rb#L23, while it has files with the necessary media types (https://github.com/pulibrary/figgy/tree/master/spec/fixtures/av/la_c0652_2017_05_bag/data), it does not fulfill the `(parent.try(:media_resource?) || parent.try(:image_resource?))` condition in https://github.com/pulibrary/figgy/blob/master/app/characterization_services/mediainfo_characterization_service.rb#L49

Reply:
Maybe stubbed characterization is causing it?

Reply:
I've restored this, I suspect that it may have been https://github.com/pulibrary/figgy/pull/3140/files#diff-2b6680eb7380002602ad9e9f157a6942R49 which was needed.